api/v1alpha1/gcpchaos_types.go

```diff
@@ -73,7 +73,7 @@ type GcpChaosSpec struct {
 	// The device name of the disk to detach.
 	// Needed in disk-loss.
 	// +optional
-	DeviceName *string `json:"deviceName,omitempty"`
+	DeviceName *[]string `json:"deviceName,omitempty"`
```

Comment:
since it changes to the array, how about change the name to `DeviceNames`

Reply:
Ok, let me modify it.